pkgs/applications/system/linux-enable-ir-emitter/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, makeWrapper
+, fetchFromGitHub
+, fetchurl
+, meson
+, ninja
+, pkg-config
+, python3
+, opencv
+, usbutils
+}:
+stdenv.mkDerivation rec {
+  pname = "linux-enable-ir-emitter";
+  version = "4.5.0";
+
+  src = fetchFromGitHub {
+    owner = "EmixamPP";
+    repo = pname;
+    rev = version;
+    hash = "sha256-Dv1ukn2TkXfBk1vc+6Uq7tw8WwCAfIcKl13BoOifz+Q=";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config makeWrapper ];
```

Comment:
Could you make the list one dependency per line? It will make diffs nicer to read and `git blame` easier to use.